tanner/tests/test_dorks_manager.py

```diff
@@ -25,14 +25,14 @@ def setUp(self):
         self.expected_result = None
 
     def test_push_init_dorks(self):
+        self.redis_client.sadd = AsyncMock()
 
         async def test():
             await self.handler.push_init_dorks(config.TannerConfig.get('DATA', 'dorks'), self.handler.dorks_key,
                                                self.redis_client)
-            self.returned_result = await self.redis_client.smembers(self.handler.dorks_key)
 
         self.loop.run_until_complete(test())
-        self.assertEqual(len(self.returned_result), 3562)
+        assert self.redis_client.sadd.called
```

Comment:
Don't mock library methods.

Reply:
I used this to just convert it into a asyncmock object to test if `sadd` is called.

Reply:
I understand that but I prefer the previous version of tests.

Reply:
I understood that but I prefer the previous version.

Reply:
Can you try passing a custom pickle with a few data samples?